### PR TITLE
Temporary fix - change URL in privacy banner

### PR DESCRIPTION
**Pull request by LilyL0u**
Resolves JIRA [[1509]](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1509)

Slack conversation explaining the issue: https://bbc-tpg.slack.com/archives/C03RA53JX0D/p1733498596613829

More context on the Jira ticket: 'DPO team need the ability to trigger an update to users when they update the privacy policy. 

There are mechanisms within Orbit & WebCore to trigger this banner when the policy is updated but this is all triggered manually by the DPO team informing us to trigger the banner. 

In general this privacy policy update notification should only be shown to a user once, at the point the policy is updated. I can share how this is achieved in webcore/orbit if helpful. 

The DPO team would like this banner to be removed but retain the right to trigger it when the policy is updated in the future. One short term fix would be to update the link to point to https://www.bbc.co.uk/usingthebbc/privacy-policy/  as mentioned above. At least then the info would be correct even if the banner is showing in error. '

Overall changes
======

'One short term fix would be to update the link to point to https://www.bbc.co.uk/usingthebbc/privacy-policy/  as mentioned above. At least then the info would be correct even if the banner is showing in error. '

The changed URL affects both canonical and amp privacy banners as they both use the rejectUrl value from the config. It would be possible to provide different URLs for canonical and amp if we wanted to, by adding a new value in the config.

Code changes
======

Service config files have updated URL: 
'https://www.bbc.com/usingthebbc/your-data-matters'.-> 'https://www.bbc.co.uk/usingthebbc/privacy-policy/'

Testing
======
1. _List the steps used to test this PR._

Check manually that all the services have the new URL. The one exception is cymrufyw.ts which has a translated page with a different URL 'rejectUrl: 'https://www.bbc.co.uk/usingthebbc/privacy-policy/cy','

Unit tests have updated snapshots.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/lib/config/services/afaanoromoo.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ export const service: DefaultServiceConfig = {
           },
           accept: 'TOLE',
           reject: 'Maaltu akka jijjirame bari',
-          rejectUrl: 'https://www.bbc.com/usingthebbc/your-data-matters',
+          rejectUrl: 'https://www.bbc.co.uk/usingthebbc/privacy-policy/',
```
Comment (karinathomasbbc) [inline src/app/lib/config/services/afaanoromoo.ts]: This and all other 40 WS services (apart from news/sport etc) should use .com 
```suggestion
          rejectUrl: 'https://www.bbc.com/usingthebbc/privacy-policy/',
```

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

File: src/app/lib/config/services/afaanoromoo.ts (hunk 1 above)
Comment (LilyL0u) [reply src/app/lib/config/services/afaanoromoo.ts]: Good spot, thanks - will change

File: src/app/lib/config/services/uzbek.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ const defaultCyrillicConfig = {
         },
         accept: 'Маъқул',
         reject: 'Нималар ўзгарди?',
-        rejectUrl: 'https://www.bbc.com/usingthebbc/your-data-matters',
+        rejectUrl: 'https://www.bbc.com/usingthebbc/privacy-policy/',
```
Comment (karinathomasbbc) [inline src/app/lib/config/services/uzbek.ts:149]: Do we need an additional entry for lat?

File: src/app/lib/config/services/zhongwen.ts [hunk 3]
```diff
@@ -239,7 +239,7 @@ export const service: ZhongwenConfig = {
           },
           accept: '可以',
           reject: '了解更新内容',
-          rejectUrl: 'https://www.bbc.com/usingthebbc/your-data-matters',
+          rejectUrl: 'https://www.bbc.com/usingthebbc/privacy-policy/',
```
Comment (karinathomasbbc) [inline src/app/lib/config/services/zhongwen.ts:242]: As for serbian - do we need another entry for simp (or trad? can't remember which one)

Comment (LilyL0u) [reply src/app/lib/config/services/zhongwen.ts:242]: Yes and weirdly I thought I had already done this. Like I actually remember checking and doing it 😆 

Will add for zhongwen and uzbek now

**Review by karinathomasbbc — APPROVED** [review]
Nice one, thanks for making alllll these changes @LilyL0u!

**Review by eagerterrier — APPROVED** [review]
T-Shirt-Image-2_1-1

**Pull request merged**